pkgs/data/themes/whitesur-kde/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, gitUpdater
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "whitesur-kde";
+  version = "unstable-2023-08-15";
+
+  src = fetchFromGitHub {
+    owner = "vinceliuice";
+    repo = finalAttrs.pname;
+    rev = "d50bc20b2b78705bb9856204066affb763fa8a35";
+    hash = "sha256-oG6QT4VQpBznM+gvzdiY4CldOwdHcBeHlbvlc52eFuU=";
+  };
+
+  postPatch = ''
+    patchShebangs install.sh
+
+    substituteInPlace install.sh \
+      --replace '$HOME/.config' $out/share \
+      --replace '$HOME/.local' $out \
+      --replace '"$HOME"/.Xresources' $out/doc/.Xresources
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/doc
+
+    name= ./install.sh
```

Comment:
What's this for?

Reply:
`.Xresourses` is installed into `$out/doc`, so that the user can copy it to its home directory if desired. `$out/doc` is not created by the install script. Therefore it should be created explicitly here.

`name ./install.sh` runs the install script. The environment variable `name` (whose value is the package name defined by the nix expression) is set to the empty string. This is needed because the install script uses the environment variable `name` as an indication of the theme name, and we want the default theme name.

Reply:
@matthiasbeyer you did not say what you want to be changed. What is it?